<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>641</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>91</width>
      <height>336</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_3">
       <property name="styleSheet">
        <string notr="true">font: 15pt &quot;隶书&quot;;</string>
       </property>
       <property name="text">
        <string>新书入库</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="pushButton_5">
       <property name="styleSheet">
        <string notr="true">font: 15pt &quot;隶书&quot;;</string>
       </property>
       <property name="text">
        <string>还书业务</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_2">
       <property name="styleSheet">
        <string notr="true">font: 15pt &quot;隶书&quot;;</string>
       </property>
       <property name="text">
        <string>删除书籍</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QPushButton" name="pushButton_6">
       <property name="styleSheet">
        <string notr="true">font: 15pt &quot;隶书&quot;;</string>
       </property>
       <property name="text">
        <string>用户注册</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QPushButton" name="pushButton_9">
       <property name="styleSheet">
        <string notr="true">font: 15pt &quot;隶书&quot;;</string>
       </property>
       <property name="text">
        <string>注销用户</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="pushButton_7">
       <property name="styleSheet">
        <string notr="true">font: 15pt &quot;隶书&quot;;</string>
       </property>
       <property name="text">
        <string>书籍检索</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="pushButton_4">
       <property name="styleSheet">
        <string notr="true">font: 15pt &quot;隶书&quot;;</string>
       </property>
       <property name="text">
        <string>借书业务</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">
font: 15pt &quot;隶书&quot;;
</string>
       </property>
       <property name="text">
        <string>馆藏浏览</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_8">
       <property name="styleSheet">
        <string notr="true">font: 15pt &quot;隶书&quot;;</string>
       </property>
       <property name="text">
        <string>用户查询</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QPushButton" name="pushButton_10">
       <property name="styleSheet">
        <string notr="true">font: 15pt &quot;隶书&quot;;</string>
       </property>
       <property name="text">
        <string>借阅记录</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>-2</y>
      <width>789</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/D:/timg.jpg);
color: rgb(255, 76, 6);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;strong&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'方正小篆体'; font-size:36pt;&quot;&gt;泽江大学图书管理系统&lt;/span&gt;&lt;/p&gt;&lt;/strong&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>[未登录]</string>
    </property>
   </widget>
   <zorder>gridLayoutWidget</zorder>
   <zorder>label</zorder>
   <zorder>tableView</zorder>
   <zorder>pushButton_11</zorder>
   <zorder>label_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>757</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
